docs/File-Formats.md

```diff
@@ -37,7 +37,7 @@ This file contains metadata about the cancer study. The file contains the follow
 3. **name**: The name of the cancer study, e.g., "Breast Cancer (Jones Lab 2013)".
 4. **description**: A description of the cancer study, e.g., "Comprehensive profiling of 103 breast cancer samples. Generated by the Jones Lab 2013". This description may contain one or more URLs to relevant information.
 5. **citation (Optional)**: A relevant citation, e.g., "TCGA, Nature 2012".
-6. **pmid (Optional)**: A relevant pubmed id. If used, the field citation has to be filled, too.
+6. **pmid (Optional)**: One or more relevant pubmed ids (comma separated). If used, the field citation has to be filled, too.
```

Comment:
I think it would be nice if we add here that white spaces are not allowed, like`(comma separated, without embedded whitespaces)`.

Reply:
done